src/actions/editor.js

```diff
@@ -198,6 +246,22 @@ export function sendDataComplete(json, action) {
     }
 }
 
+export function sendRecurringData(formValues, contentLanguages, user, updateExisting = false, publicationStatus, superEventId) {
+    return (dispatch) => {
+        const subEvents = Object.assign({}, formValues.sub_events)
+        const baseEvent = Object.assign({}, formValues, { sub_events: {}, super_event: {'@id': superEventId}})
```

Comment:
Here, actually we would want to post a base event that depicts the *whole* event series, with start time as start time of the first event and end time as the end time of the last event. All the other fields may be the same as in all the sub events.

Then, we would post all the individual events (including the first one) as a batch, referring to the abstract base event as being their super.

This way, the base event depicts the whole event series taking place for the whole duration of the repeats. All of the actual events (including the first one) will be sub events to the base event.

There is no need to display the base element in the UI as such, and it is fine to redirect the first created event in the UI after creating the series. Still, in the API the base event should be separate from any of the actual event instances. It can be distinguished later (and filtered out if we don't want it to be displayed in the event listing) by the super event type = recurring field.